pkg/apis/kubeone/types.go

```diff
@@ -63,6 +63,15 @@ type KubeOneCluster struct {
 	AssetConfiguration AssetConfiguration `json:"assetConfiguration,omitempty"`
 	// RegistryConfiguration configures how Docker images are pulled from an image registry
 	RegistryConfiguration *RegistryConfiguration `json:"registryConfiguration,omitempty"`
+	// KubeletConfiguration configures the kubelet
+	KubeletConfiguration KubeletConfiguration `json:"kubeletConfiguration"`
+}
+
+// KubeletConfiguration
+type KubeletConfiguration struct {
```

Comment:
I don't like that this struct is called `KubeletConfiguration` because we already have the `KubeletConfig` struct used for something else. Maybe we can call it `LoggingConfiguration`? @kron4eg WDYT?

Reply:
Indeed, https://docs.kubermatic.com/kubeone/master/references/kubeone_cluster_v1beta2/#kubeletconfig
and we should _use that_ struct to configure log size

Reply:
@kron4eg and I discussed this and we can't use this struct because it's used for per-host settings, and we don't want to configure logging that way. @nerdeveloper Can you just rename this struct to `LoggingConfig`?

Reply:
Sure